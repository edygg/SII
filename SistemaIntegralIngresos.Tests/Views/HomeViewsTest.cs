using System;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using ApprovalTests.Reporters;
using SistemaIntegralIngresos.Controllers;
using ApprovalTests.Asp.Mvc;

namespace SistemaIntegralIngresos.Tests.Views
{
    [TestClass]
    [UseReporter(typeof(FileLauncherReporter))]
    public class HomeViewsTest : MvcTest
    {
        [TestMethod]
        public void IndexHomeViewTest()
        {
            MvcApprovals.VerifyMvcPage(new HomeController().Index);
        }

        [TestMethod]
        public void ContactHomeViewTest()
        {
            MvcApprovals.VerifyMvcPage(new HomeController().Contact);
        }

        [TestMethod]
        public void AboutHomeViewTest()
        {
            MvcApprovals.VerifyMvcPage(new HomeController().About);
        }
    }
}
