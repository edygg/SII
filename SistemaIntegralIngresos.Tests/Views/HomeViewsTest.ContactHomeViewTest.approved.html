<!DOCTYPE html>
<html lang="en">
    <head><base href="http://localhost:19610/Home">
        <meta charset="utf-8" />
        <title> | SII</title>
        <link href="/favicon.ico" rel="shortcut icon" type="image/x-icon" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <link href="/Content/foundation.min.css" rel="stylesheet"/>

        <link href="/Content/Layout.css" rel="stylesheet"/>

        <link href="/Content/foundation-icons/foundation-icons.css?v=-6_zzR_BwLqFqCPKpXxa56eDHYlTBHOUql5yYn8-ikw1" rel="stylesheet"/>

        <script src="/Scripts/modernizr-2.6.2.js"></script>

    </head>
    <body>
        <header>
            <nav class="top-bar" data-topbar role="navigation">
    <ul class="title-area">
        <li class="name">
            <h1><a href="/">Inicio</a></h1>
        </li>
        <li class="toggle-topbar menu-icon"><a href="#"><span>Menu</span></a></li>
    </ul>

    <section class="top-bar-section">
        <!-- Right Nav Section -->
        <ul class="right">
        </ul>
    </section>
</nav>
        </header>
        <section id="body">
            <header>
                <div class="row">
    <div class="medium-10 small-9 columns text-center">
        <h1>Sistema Integral de Ingresos (<acronym title="Sistema Integral de Ingresos">SII</acronym>)</h1>
        <h2>Universidad Tecnológica Centroamericana</h2>
    </div>
    <div class="medium-2 small-3 columns">
        <img src="/Images/unitec-logo.png" alt="Unitec Logo" />
    </div>
</div>
<br />
<div class="row">
    <div class="text-center">
        <h3></h3>
    </div>
</div>
            </header>
            

<hgroup class="title">
    <h1>Contact.</h1>
    <h2></h2>
</hgroup>

<section class="contact">
    <header>
        <h3>Phone</h3>
    </header>
    <p>
        <span class="label">Main:</span>
        <span>425.555.0100</span>
    </p>
    <p>
        <span class="label">After Hours:</span>
        <span>425.555.0199</span>
    </p>
</section>

<section class="contact">
    <header>
        <h3>Email</h3>
    </header>
    <p>
        <span class="label">Support:</span>
        <span><a href="mailto:Support@example.com">Support@example.com</a></span>
    </p>
    <p>
        <span class="label">Marketing:</span>
        <span><a href="mailto:Marketing@example.com">Marketing@example.com</a></span>
    </p>
    <p>
        <span class="label">General:</span>
        <span><a href="mailto:General@example.com">General@example.com</a></span>
    </p>
</section>

<section class="contact">
    <header>
        <h3>Address</h3>
    </header>
    <p>
        One Microsoft Way<br />
        Redmond, WA 98052-6399
    </p>
</section>
        </section>

        <script src="/Scripts/vendor/jquery.js"></script>

        <script src="/Scripts/vendor/jquery.cookie.js"></script>

        <script src="/Scripts/vendor/fastclick.js"></script>

        <script src="/Scripts/vendor/modernizr.js"></script>

        <script src="/Scripts/vendor/placeholder.js"></script>

        <script src="/Scripts/foundation.min.js"></script>

        <script src="/Scripts/Application.js"></script>
;
        
    </body>
</html>
